core/templates/dev/head/css/oppia.css

```diff
@@ -253,6 +254,26 @@ textarea {
 }
 
 /*
+  Styles for the site feedback form.
+*/
+.oppia-site-feedback {
+  bottom: 5px;
+  color: #eee;
+  cursor: pointer;
+  left: 0;
+  opacity: 0.6;
+  padding: 4px 8px;
+  position: fixed;
+}
+.oppia-site-feedback:hover,
+.oppia-site-feedback:focus,
+.oppia-site-feedback:active {
```

Comment:
I recommend this CSS:

  color: rgba(0,0,0,0.87);
  opacity: 1;
  text-decoration: none;